benchmarks/activesetbench.c

```diff
@@ -55,7 +55,7 @@ int main(int argc, char *argv[]) {
     d2 = getTimeMillis();
 
     printf("time: %d (ms)\tthroughput: %.2f (millions ops/sec)\t", (int)(d2 - d1), bench_args.runs * bench_args.nthreads / (1000.0 * (d2 - d1)));
-    printStats(bench_args.nthreads);
+    printStats(bench_args.nthreads, bench_args.total_runs);
```

Comment:
Here (and in all files) you could just pass a const pointer to BenchArgs struct, also a way to make future proof.

Reply:
The functionality provided by stats.c and stats.h is provided by the runtime and I think should not be strongly depended on BenchArgs, which is functionality used only by the benchmarks and not the runtime. However, we may re-consider it in the next release.

Reply:
I tested in OpenSUSE 15.2 over windows 20H2 + wsl1 and it works fine. Thus, I think check this point as resolved.

Reply:
Fine by me